entity-framework/core/extensions/index.md

```diff
@@ -149,3 +119,33 @@ EntityFrameworkCore extensions for Bulk operations (Insert, Update, Delete).
 Adds design-time pluralization to EF Core.
 
 [GitHub repository](https://github.com/bricelam/EFCore.Pluralizer)
+
+### PomeloFoundation/Pomelo.EntityFrameworkCore.Extensions.ToSql
+
+Get the generated SQL.
```

Comment:
> Implements a simple extension method that be used to obtain SQL generated by EF Core for a LINQ query. Note that the functionality of the method is limited because EF Core can generate more than one SQL statement for a single LINQ query, and different SQL statements depending on parameter values.